python/ray/data/read_api.py

```diff
@@ -1326,6 +1327,58 @@ def convert(ds: "datasets.Dataset") -> Dataset[ArrowRow]:
         )
 
 
+@PublicAPI
+def from_tf(
```

Comment:
Can we expand this? 
```suggestion
def from_tensorflow(
```

Reply:
We have a method called `Dataset.to_tf`. I prefer `tensorflow` over `tf`, but we should be consistent. 

What do you think? We could always rename `Dataset.to_tf` to `Dataset.to_tensorflow`.

Reply:
Ah right... Maybe we can:
1. Name this `from_tensorflow`.
2. Create a `to_tensorflow`.
3. Have `to_tf` redirect to `to_tensorflow` and eventually deprecate it.

But will leave to folks like @clarkzinzow @c21 to make the call here.

Reply:
I'd actually vote for renaming these to `from_tf_dataset`, `to_tf_dataset`, `from_torch_dataset`, `to_torch_dataset`, etc. This is more explicit, discoverable, and better matches what the rest of the ecosystem is doing, e.g. HuggingFace: https://huggingface.co/docs/datasets/v2.6.1/en/package_reference/main_classes#datasets.Dataset.to_tf_dataset

I think that `tf` is a common enough shorthand that we shouldn't expand it to `tensorflow`, but happy to hear others' opinions, and maybe we can do a quick ecosystem check to verify this.